index.html

```diff
@@ -1,234 +1,216 @@
-                <li><a href="https://github.com/alexisuwimana" class=""><img class="social-twitter" src="images/social/Twitter.png" alt="twitter"></a></li>   
-                <li><a href="https://github.com/alexisuwimana" class=""><img class="social-M" src="images/social/M-Disabled.png" alt="M"></a></li>   
-            </ul>
+    <div class="overall-container">
+      <section class="main-container">
+        <div class="intro-header">
+          <h1>
+            Hey There. <br />
+            I’m Alexis
+          </h1>
+          <div class="intro-detail">
+            <h2 class="intro-title">I’am a Software Developer</h2>
+            <p>
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don’t hestiate to contact me.
+            </p>
+          </div>
         </div>
-    </section> 
-
-        <!-- works -->
-
-    <section class="Works">
-        <div class="works-title" >
-            <h3 class="heading-3">My Recent Works </h3> 
-            <hr class="works-line">
+        <div class="social">
+          <ul class="social-ul">
+            <li>
+              <a href="#"><i class="fab fa-github" title="github icon"></i></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fab fa-linkedin-in" title="linkedin icon"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fab fa-angellist" title="Angel icon"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fab fa-twitter" title="twitter icon"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fab fa-medium-m" title="Medium icon"></i
+              ></a>
+            </li>
+          </ul>
         </div>
-        <div class="works-img">
-            <img src="./images/works/img-works-0.svg" alt="works">
+      </section>
+      <section class="my-projects-section">
+        <div id="projects-title">
+          <h1>My Recent Works</h1>
         </div>
-        <div class="stories">
-            <h1 class="stories-headings">Multi-Post Stories</h1>
-            <p class="stories-content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="inner-job-item-two">
         </div>
-    
-        <div class="Stories-lang">
-            <ul class="stories-lang-list">
-                <li class="stories-list-item">css</li>
-                <li class="stories-list-item">html</li>
-                <li class="stories-list-item">bootstrap</li>
-                <li class="stories-list-item">Ruby</li>
-            </ul>
```

Comment:
- [ ]  It may be a good idea to don't remove your Featured project because it's a mandatory part that you need to show following the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) template. 😉 

![Feature project mobile](https://user-images.githubusercontent.com/79876959/139561069-ebe38f7c-44a6-49c2-a354-720d2887bf28.PNG)
![Feature project desktop](https://user-images.githubusercontent.com/79876959/139561073-f25be89f-5b59-4c2c-9c61-83382518ba2c.PNG)